client/client-multi/client-java-core/src/main/java/org/kaaproject/kaa/client/channel/impl/channels/CoapChannel.java

```diff
@@ -0,0 +1,265 @@
+package org.kaaproject.kaa.client.channel.impl.channels;
```

Comment:
This file needs Apache license header. Both Jenkins and Travis are currently failing because of missing Apache license headers.